lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -255,9 +255,11 @@ def call(params)
                                              do_not_touch_users_that_are_not_mentioned: do_not_touch_users_that_are_not_mentioned)
                       end
 
+                      params.gdc_logger.debug("Working client ids are: #{working_client_ids.join(', ')}")
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)